docs/installation-and-operations/installation/packaged/README.md

```diff
@@ -28,6 +28,7 @@ The package is available for the following Linux distributions:
 
 | Distribution (64 bits only)                 |
 | ------------------------------------------- |
+| [Ubuntu 22.04 Focal](#ubuntu-2204)          |
```

Comment:
@crohr Is "Focal" still the right name? Here it is called "Jammy Jellyfish" https://releases.ubuntu.com/jammy/

Reply:
thanks for spotting that